vj4/template.py

```diff
@@ -42,7 +42,9 @@ def __init__(self):
 MARKDOWN_EXTENSIONS = (hoedown.EXT_TABLES |  # Parse PHP-Markdown style tables.
                        hoedown.EXT_FENCED_CODE |  # Parse fenced code blocks.
                        hoedown.EXT_AUTOLINK |  # Automatically turn safe URLs into links.
-                       hoedown.EXT_NO_INTRA_EMPHASIS)  # Disable emphasis_between_words.
+                       hoedown.EXT_NO_INTRA_EMPHASIS |  # Disable emphasis_between_words.
+                       hoedown.EXT_MATH |  # Allow math exp
+                       hoedown.EXT_MATH_EXPLICIT )  # Explicitly inline/block
```

Comment:
\# Instead of guessing by context, parse $inline math$ and $$always block math$$ (requires --math).